addons/mail/static/src/components/messaging_menu/messaging_menu.xml

```diff
@@ -5,8 +5,8 @@
         <li class="o_MessagingMenu" t-att-class="{ show: messagingMenu.isOpen, 'o-mobile': env.messaging.device.isMobile }">
             <!-- TODO FIXME restore mobile stuff here (activeMobileThread) -->
             <t t-if="!env.messaging.device.isMobile or true">
-                <a class="o_MessagingMenu_toggler o-no-caret" t-att-class="{ show: messagingMenu.isOpen }" href="#" t-on-click="_onClickToggler" data-toggle="dropdown" data-display="static" title="Conversations" role="button" aria-expanded="false">
-                    <i class="o_MessagingMenu_icon fa fa-comments-o" role="img" aria-label="Messages"/>
+                <a class="o_MessagingMenu_toggler o-no-caret" t-att-class="{ show: messagingMenu.isOpen, o_MessagingMenu_togglerNoNotification: !messagingMenu.counter }" href="#" t-on-click="_onClickToggler" data-toggle="dropdown" data-display="static" title="Conversations" role="button" aria-expanded="false">
```

Comment:
newly introduced classname looks like a modifier, therefore I would expect it to follow our naming convention for modifiers (`o-<modifierName>` or `o_<ComponentName>_<partName>-<modifierName>`).

Reply:
`o-noNotification`?

Reply:
`o-noNotification` or `o-no-notification`, both are good (in fact we seem to have used `<modifier-name>`, which is ok with shorter version but not so longer one).

So let's take `o-no-notification` to be consistent right now.

Reply:
ok